core/domain/config_domain.py

```diff
@@ -222,3 +222,25 @@ def update_moderator_ids():
     SET_OF_STRINGS_SCHEMA,
     'Banned usernames (editing permissions for these users have been removed)',
     [])
+
+
+# TODO(milit): Check the language codes are valid
```

Comment:
This check is going to be hard to do with the current infrastructure. I would suggest just making this a constant in feconf for now, and having a backend test that checks that the contents of this constant correspond to valid language codes. We'll generalize it later, when the need arises. I have a feeling that the right way to generalize might actually be to add a boolean flag saying "enable multiple languages", but I think we need more use cases to really understand what to offer here.